lib/utils.js

```diff
@@ -43,7 +44,29 @@ function isRedisReady(client) {
   });
 }
 
+/**
+ * Checks the size of string for ascii/non-ascii characters
+ * (Reference: https://stackoverflow.com/a/23318053/1347170)
+ * @param {string} str
+ */
+function lengthInUtf8Bytes(str) {
+  const m = encodeURIComponent(str).match(/%[89ABab]/g);
```

Comment:
is this better/faster than using ```Buffer.byteLength``` ?
https://stackoverflow.com/questions/9864662/how-to-get-the-string-length-in-bytes-in-nodejs

Reply:
I'm not sure, I'll do some tests.